src/main/java/org/zone/commands/structure/region/flags/interact/itemframe/ZoneFlagInteractItemframesViewCommand.java

```diff
@@ -0,0 +1,72 @@
+package org.zone.commands.structure.region.flags.interact.itemframe;
+
+import net.kyori.adventure.identity.Identity;
+import net.kyori.adventure.text.Component;
+import org.jetbrains.annotations.NotNull;
+import org.spongepowered.api.command.CommandResult;
+import org.zone.Identifiable;
+import org.zone.commands.system.ArgumentCommand;
+import org.zone.commands.system.CommandArgument;
+import org.zone.commands.system.arguments.operation.ExactArgument;
+import org.zone.commands.system.arguments.operation.OptionalArgument;
+import org.zone.commands.system.arguments.zone.ZoneArgument;
+import org.zone.commands.system.context.CommandContext;
+import org.zone.region.Zone;
+import org.zone.region.flag.FlagTypes;
+import org.zone.region.flag.interact.itemframe.InteractItemframesFlag;
+import org.zone.region.group.key.GroupKeys;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * Used to view the details of {@link InteractItemframesFlag}
+ */
+public class ZoneFlagInteractItemframesViewCommand implements ArgumentCommand {
+
+    public static final ExactArgument REGION = new ExactArgument("region");
+    public static final ExactArgument FLAG = new ExactArgument("flag");
+    public static final ZoneArgument ZONE_VALUE = new ZoneArgument("zone_value", new ZoneArgument
+            .ZoneArgumentPropertiesBuilder()
+            .setLevel(GroupKeys.INTERACT_ITEMFRAME));
```

Comment:
This should not check the level. If it doesnt have the flag then it should say that